jekyll/_cci2/orb-author-validate-publish.md

```diff
@@ -1,107 +1,45 @@
-
-### orb-tools/increment
-
-This command uses the CLI to increment the version of an orb in the registry. If the orb does not have a version yet it starts at 0.0.0. The following parameters may be passed with this command:
-
-Parameter | Description
-------------|-----------
-orb-path | Path to an orb file.
-orb-ref | A version-less orb-ref in the form /
-segment | The semantic version segment to increment ‘major’ or ‘minor’ or ‘patch’
-publish-token-variable | The env var containing your token. Pass this as a literal string such as $ORB_PUBLISHING_TOKEN. Do not paste the actual token into your configuration. If omitted it’s assumed the CLI has already been setup with a valid token.
-validate | Boolean for whether or not to do validation on the orb. Default is false.
-checkout | Boolean for whether or not to checkout as a first step. Default is true.
-attach-workspace | Boolean for whether or not to attach to an existing workspace. Default is false.
-workspace-root | Workspace root path that is either an absolute path or a path relative to the working directory. Defaults to ‘.’ (the working directory)
-{: class="table table-striped"}
-
-### orb-tools/publish
-
-This command is used to publish an orb. The following parameters may be passed with this command:
+1. If you have not already done so, claim a namespace for your user/organization using the following command, substituting your namespace choice and GitHub org name:
+```sh
+circleci namespace create <my-namespace> github <my-gh-org>
+```
+**Note:** When creating a namespace via the CircleCI CLI, be sure to specify the VCS provider.
 
-Parameter | Description
-------------|-----------
-orb-path | Path of the orb file to publish.
-orb-ref | A full orb-ref in the form of /@
-publish-token-variable | The env var containing your publish token. Pass this as a literal string such as $ORB_PUBLISHING_TOKEN. DO NOT paste the actual token into your configuration. If omitted it’s assumed the CLI has already been setup with a valid token.
-validate | Boolean for whether or not to do validation on the orb. Default is false.
-checkout | Boolean for whether or not to checkout as a first step. Default is true.
-attach-workspace | Boolean for whether or not to attach to an existing workspace. Default is false.
-workspace-root | Workspace root path that is either an absolute path or a path relative to the working directory. Defaults to ‘.’ (the working directory)
-{: class="table table-striped"}
+1. Create your orb inside your namespace. At this stage no orb content is being generated, but the naming is reserved for when the orb is published:
+```sh
+circleci orb create <my-namespace>/<my-orb-name>
+```
 
-### Validate and Publish Example
+1. Create the content of your orb in a YAML file. Generally, you would do this in your code editor, in a git repo made for your orb, but, For this simple example, let's assume a file in `/tmp/orb.yml` could be made with a bare-bones orb like:
```

Comment:
```suggestion
1. Create the content of your orb in a YAML file. For example:
```yaml
version: 2.1
description: A greeting command orb
commands:
    greet:
        description: Greet someone with a "hello".
        parameters:
            to:
                type: string
                default: World
        steps:
            - run: echo "Hello, << parameters.to >>"
```
```